### DISCUSSION TITLE
Do I always need documentation in the default language to support i18?

### DISCUSSION THREAD

**Discussion by dninomiya** [facebook/docusaurus#7040] [Q&A] [answered]
For example, on a site where ja (default) and en are enabled, is it possible to enable only the `en` version of the document? Currently it doesn't seem to work without both `ja` and` en` documentation.

**Comment by Josh-Cena** [comment, marked as answer]
No—we expect the routes structure to be exactly isomorphic between locales. If you have a page with path `/docs/intro`, we will make sure that `/en/docs/intro` also exists, and vice versa. Not doing so may result in weird bugs like #5844.

**Reply by dninomiya** [reply]
Thank you for your quick answer!
